checkout-extension/shopify.extension.toml.liquid

```diff
@@ -1,35 +1,51 @@
 # Learn more about configuring your checkout UI extension:
 # https://shopify.dev/api/checkout-extensions/checkout/configuration
 
+# The version of APIs your extension will receive. Learn more:
+# https://shopify.dev/docs/api/usage/versioning
 api_version = "2023-07"
 
 [[extensions]]
-name = "{{ name }}"
 type = "ui_extension"
+name = "{{ name }}"
 handle = "{{ handle }}"
 
-# [[extensions.metafields]]
-# namespace = "my-namespace"
-# key = "my-key"
-# [[extensions.metafields]]
-# namespace = "my-namespace"
-# key = "my-other-key"
+# Controls where in Shopify your extension will be injected,
+# and the file that contains your extension’s source code. Learn more:
+# https://shopify.dev/docs/api/checkout-ui-extensions/unstable/targets-overview
+
+[[extensions.targeting]]
+module = "./src/Checkout.{{ srcFileExtension }}"
+target = "purchase.checkout.block.render"
 
 [extensions.capabilities]
+# Gives your extension access to directly query Shopify’s storefront API.
+# https://shopify.dev/docs/api/checkout-ui-extensions/unstable/configuration#api-access
 api_access = true
-network_access = true
-block_progress = true
```

Comment:
I removed this since it's not something we want or expect every extension doing.